tools-local/tasks/project.json

```diff
@@ -22,7 +24,7 @@
         "portable-net45+win8+wpa81"
       ]
     },
-    "net45": {
+    "net451": {
```

Comment:
Changed.  This was also wrong in `tools-local\Microsoft.DotNet.Build.Tasks.Local\project.json`.